proposals/description/fip_41.ts

```diff
@@ -0,0 +1,71 @@
+import { ProposalDescription } from '@custom-types/types';
+
+const fip_41: ProposalDescription = {
+  title: 'FIP-41: LUSD Auction',
+  commands: [
+    {
+      target: 'fei',
+      values: '0',
+      method: 'mint(address,uint256)',
+      arguments: ['{feiDAOTimelock}', '1100000000000000000000000'],
+      description: 'Mint 1.1M FEI for swap'
+    },
+    {
+      target: 'fei',
+      values: '0',
+      method: 'approve(address,uint256)',
+      arguments: ['{saddleD4Pool}', '1100000000000000000000000'],
+      description: 'Approve 1.1M FEI for swap'
+    },
+    {
+      target: 'saddleD4Pool',
+      values: '0',
+      method: 'swap(uint8,uint8,uint256,uint256,uint256)',
+      arguments: [
+        '1', // tokenIndexFrom: FEI
+        '3', // tokenIndexTo: LUSD
+        '1100000000000000000000000', // dx
+        '1089000000000000000000000', // minDy
+        '1640995200' // deadline: 2022-01-01
+      ],
+      description: 'Swap 1.1M FEI for LUSD'
+    },
+    {
+      target: 'lusd',
+      values: '0',
+      method: 'transfer(address,uint256)',
+      arguments: ['{feiLusdLBPSwapper}', '1089000000000000000000000'],
```

Comment:
we'll need to make sure we submit the dao-execute transaction via flashbots protect, or someone could grief this swap